roles/openshift_node_upgrade/meta/main.yml

```diff
@@ -0,0 +1,12 @@
+galaxy_info:
+  author: your name
+  description: OpenShift Node upgrade
+  company: Red Hat, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 2.1
+  platforms:
+  - name: EL
+    versions:
+    - 7
+dependencies:
+  - role: openshift_common
```

Comment:
Arguably openshift_version is a dep here as well, which basically just no-ops if it sees that the three core variables are already set.